lib/cequel/metal/keyspace.rb

```diff
@@ -266,6 +287,7 @@ def client_without_keyspace
       def client_options
         {hosts: hosts, port: port}.tap do |options|
           options.merge!(credentials) if credentials
+          options[:server_cert] = server_cert if server_cert
```

Comment:
It looks like `server_cert` is the only one of these configuration keys that we actually use anywhere?

At a higher level, I don't think it makes sense to have all of these as top-level values in the configuration. After all, it only makes sense to have `:server_cert`, `:private_key`, etc. if `:ssl` is `true`. I would propose rolling all the SSL-related values up into a single object (a `Struct` would probably do fine), and then failing fast if `:ssl` is set to true but any of the SSL-related properties are missing.

Note that this doesn't necessarily mean changing the interface to `#configure`; I'm not strongly opposed to having these as top-level keys in the options passed to that method, just as top-level instance variables in the `Keyspace` itself.